internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,55 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	pd "github.com/Yandex-Practicum/tracker/internal/personaldata"
+	se "github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	pd.Personal
 }
 
+var errData = errors.New("incorrect data")
+
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	s := strings.Split(datastring, ",")
+	if len(s) != 2 {
+		return errData
```

Comment:
Лучше более информативно писать ошибки
```suggestion
		fmt.Errorf("length of data must be 2")
```